Makefile

```
@@ -19,3 +19,6 @@ check:
 
 test:
 	@PYTHONDONTWRITEBYTECODE=1 poetry run python3 -m unittest discover tests/ -v
+
+oi:
+	echo "oi"
```

Ops, essa parte aqui era só um teste. Não seria para ser adicionado no PR de fato:

```suggestion

oi:
	echo "oi"
```